app/README.md

```diff
@@ -1,25 +1,51 @@
+# DGRSC - Document Generation Service Showcase Backend
+DGRSC (Document Generation Service Showcase) Backend is a simple node.js wrapper around the [BC Gov. Common Document Generation Service](https://github.com/bcgov/common-document-generation-service.git).  The wrapper is a simple pass through to the CDOGS service, its main purpose is to hide the service client credentials from the frontend.  We host the showcase frontend from this node.js backend.  
 
 # Configuration
-The CDOGS Backend will require some configuration, namely credentials to CDOGS and where to call CDOGS.  We also want to lock down our api using a JWT Token for DGRSC.  We will need to configure our cdogs-backend to authenticate using the same KeyCloak realm and client as the [frontend](../frontend/README.md) uses.    
+The DGRSC Backend will require some configuration, namely credentials to CDOGS and where to call CDOGS.  We also want to lock down our api using a JWT Token for DGRSC.  We will need to configure our DGRSC backend to authenticate using the same KeyCloak realm and client that the [frontend](../frontend) uses. All configuration for the frontend will come from the DGRSC backend, fetched by calling a configuration url (/config).   
 
 We are using the npm library: [config](https://www.npmjs.com/package/config), to configure our application.  We can either set environment variables to be picked up by [custom-environment-variables.json](./config/custom-environment-variables.json) or we can create a local configuration file (do not check into source control) such as local.json.  Please read the config library documentation to see how it overlays various environment files and uses environment variables to override those values (when provided).
 
+**FRONTEND\_APIPATH** has no beginning '/', that is so it will always call the relative api.  
+
 #### Environment Variables
 
 | Name | Default | Description |  
 | --- | --- | --- |  
-| CDOGS_CLIENT_ID | CDOGS_SERVICE_CLIENT | The name of the service client in the realm that has been granted access to CDOGS.  See [GetOK](https://github.com/bcgov/nr-get-token.git) for more information |  
+| FRONTEND_KC_CLIENTID | dgrsc-frontend | The name of the frontend authorization service client - Users login to this (KeyCloak) client to get authenticated and authorized to DGRSC.  See [GetOK](https://github.com/bcgov/nr-get-token.git) for more information | 
+| FRONTEND_KC_REALM | 98r0z7rz | The KeyCloak realm id |  
+| FRONTEND_KC_SERVERURL | https://sso-dev.pathfinder.gov.bc.ca/auth | The KeyCloak authorization url |  
+| FRONTEND_APIPATH | api/v1 | Relative path for frontend to call backend api |  
+| FRONTEND_BASEPATH | /app | Base path where frontend will be hosted |  
+| CDOGS_CLIENT_ID | DGRSC_SERVICE_CLIENT | The name of the service client in the realm that has been granted access to CDOGS.  See [GetOK](https://github.com/bcgov/nr-get-token.git) for more information |  
```

Comment:
Even though the .git part of the url works correctly, it might be better to drop that part as we're expecting a hyperlink to the page, not the raw git repository (though Github handles that seamlessly).